# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1323967263 18000
#      Thu Dec 15 11:41:03 2011 -0500
# Branch stable
# Node ID 7e60182e6a1ca8a7694167cd68a8b3aedd40a737
# Parent  c3d4015624100f7122fb6ab029691795f9cb75d1
avoid deprecated Array constructor

* liboctave/regexp.cc (regexp::is_match): Use Array (const
dim_vector&), not Array (octave_idx_type, int).

diff --git a/liboctave/regexp.cc b/liboctave/regexp.cc
--- a/liboctave/regexp.cc
+++ b/liboctave/regexp.cc
@@ -412,17 +412,17 @@ regexp::is_match (const std::string& buf
   return ! match_string.empty ();
 }
 
 Array<bool>
 regexp::is_match (const string_vector& buffer)
 {
   octave_idx_type len = buffer.length ();
 
-  Array<bool> retval (len, 1);
+  Array<bool> retval (dim_vector (len, 1));
 
   for (octave_idx_type i = 0; i < buffer.length (); i++)
     retval(i) = is_match (buffer(i));
 
   return retval;
 }
 
 std::string
